Sources/NIO/ChannelPipeline.swift

```diff
@@ -109,6 +109,11 @@
 /// - 1 and 2 don't implement `ChannelOutboundHandler`, and therefore the actual evaluation order of a outbound event will be: 5, 4, and 3.
 /// - If 5 implements both `ChannelInboundHandler` and `ChannelOutboundHandler`, the evaluation order of an inbound and a outbound event could be 125 and 543 respectively.
 ///
+/// Note:   Handlers may choose not to progagate messages down the pipeline immediaitely.  For example a handler may need to wait
```

Comment:
```suggestion
/// Note:   Handlers may choose not to progagate messages down the pipeline immediately.  For example a handler may need to wait
```

Reply:
In addition to Cory's suggestion:

```suggestion
/// Note:   Handlers may choose not to propagate messages down the pipeline immediaitely.  For example a handler may need to wait
```

Reply:
This reads good, thanks @PeterAdams-A !

The spacing is a bit weird... Why double spaces everywhere?

```suggestion
Note: Handlers may choose not to progagate messages down the pipeline immediately. For example a handler may need to wait
```

etc?